src/ResourceManager/RecoveryServices.Backup/RecoveryServicesBackupCmdletBase.cs

```diff
@@ -40,6 +40,7 @@ protected void InitializeAzureBackupCmdlet()
         {
             var cloudServicesClient = AzureSession.ClientFactory.CreateClient<CloudServiceManagementClient>(DefaultContext, AzureEnvironment.Endpoint.ResourceManager);
             HydraAdapter = new HydraAdapterNS.HydraAdapter(cloudServicesClient.Credentials, cloudServicesClient.BaseUri);
+            new Logger.Logger(WriteWarning, WriteDebug, WriteVerbose);
```

Comment:
Instead of using statics, can we use an instance of the logger? I believe that it'll be a much cleaner and safer approach.